### PR TITLE
feat: Tokens page hints

### LINKED ISSUE #183: Tokens page hints
Display hints on tokens page.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #183 

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add token hins

### CODE REVIEW

File: src/utils/hints/tokensHints.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const tokenHints = {
+  tokenSymbol: 'Symbol of a token. Symbol does not have to be unique.',
```
Comment (thepiwo) [inline src/utils/hints/tokensHints.js:2]: "The symbol does not..."

File: src/utils/hints/tokensHints.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  tokenName: 'Full name of a token.',
```
Comment (thepiwo) [inline src/utils/hints/tokensHints.js:3]: the token?

File: src/utils/hints/tokensHints.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  tokenContractId: 'Identifier of a Smart Contract that implements AEX-9 interface.',
```
Comment (thepiwo) [inline src/utils/hints/tokensHints.js:4]: the smart contract? its shown in the specific location of a single identifier right?

File: src/utils/hints/tokensHints.js (hunk 1 above)
Comment (janmichek) [reply src/utils/hints/tokensHints.js:2]: Right, I will spellcheck separately next time .)

File: src/utils/hints/tokensHints.js (hunk 2 above)
Comment (janmichek) [reply src/utils/hints/tokensHints.js]: here is disputable, but I went with `the`.

File: src/utils/hints/tokensHints.js (hunk 3 above)
Comment (janmichek) [reply src/utils/hints/tokensHints.js]: Yes, here make sense. Fixed

**Review by thepiwo — APPROVED** [review]

**Review by lukeromanowicz — APPROVED** [review]
I added one optional suggestion but overall it looks good. Nice job :+1:

File: src/utils/hints/tokensHints.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const tokenHints = {
+  tokenSymbol: 'Symbol of a token. The symbol does not have to be unique.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/tokensHints.js]: even more punchy. I used that. Thanks

**Review by michele-franchi — APPROVED** [review]
Good job! 👏

**Review by CharisSiarampalis — APPROVED** [review]

**Comment by janmichek** [comment]
@marc0olo Could you check here, please?

**Pull request merged**